pkg/component/gardener/apiserver/apiserver.go

```diff
@@ -160,7 +160,6 @@ func (g *gardenerAPIServer) Deploy(ctx context.Context) error {
 	runtimeResources, err := runtimeRegistry.AddAllAndSerialize(
 		g.podDisruptionBudget(),
 		g.serviceRuntime(),
-		g.horizontalPodAutoscaler(),
```

Comment:
Could you please adapt https://github.com/gardener/gardener/blob/d839b336dfd7576f6c4aff67dc0981c4e20209b2/docs/development/autoscaling-specifics-for-components.md#L47-L49?

You could add the reasoning there why the gardener-apiserver is NOT scaled with HPA.